types/react-native/index.d.ts

```diff
@@ -1545,7 +1546,8 @@ export interface ViewStyle extends FlexStyle, TransformsStyle {
     borderTopColor?: string;
     borderTopLeftRadius?: number;
     borderTopRightRadius?: number;
-    borderTopWidth?: number
+    borderTopWidth?: number;
+    color?: string;    
     opacity?: number;
```

Comment:
@huhuanming 

Yep, i diged into the source code, turns out that you are right, this problem it's my own project types definition error, not yours.
Sorry for waste your time, i have update the commit and pr to just fix the typo i found.

Thanks.